src/app/rundown-view/components/unfocused/unfocused.component.html

```diff
@@ -0,0 +1 @@
+<span class="label"> {{ outOfFocusMessage.toUpperCase() }}</span>
```

Comment:
Extraneous whitespace between `>` and `{{`.

Reply:
Removed!